scala-package/packageTest/README.md

```diff
@@ -0,0 +1,50 @@
+# MXNet Scala Package Test
+
+This is an project created to run the test suite on a fully packaged mxnet jar.
+
+## Setup
+
+### Install Package
+
+To run the test suite, first install the package.  This can be done either by installing directly from a jar with `mvn install:install-file -Dfile=<path-to-file>` or by running `make scalainstall` in the main mxnet folder.  Note that if you use `mvn install:install-file`, you will be unable to run the example tests unless you also install the mxnetexamples jar. You can run all tests except for those examples with `make scalaintegrationtestwithoutexamples`.
+
+### Build
+
+Build the mxnet tests by running `make scalapkg` and then `make scalatestcompile` from the main mxnet directory.  This is needed for test discovery.
+
+## Run
+
+To run, ensure the versions are correct in the `Makefile`.  Then, just run `make scalaintegrationtest` to execute the test suite
+
```

Comment:
Please add more information about the `Makefile` for test packages, about the usages of the build stages.